lib/jnpr/junos/op/teddb.yml

```diff
@@ -17,6 +17,9 @@ TedView:
   link-in: { ted-database-link-in : int }
   link-out: { ted-database-link-out : int }
   protocol: ted-database-protocol
+  srgb_base: { ted-spring-capability/ted-spring-srgb-block/ted-spring-srgb-block-start : int }
```

Comment:
Please change `_` or to `-` to maintain uniformity. Also, change `srgb_flag` to `srgb_flags`